.travis.yml

```diff
@@ -59,10 +59,9 @@ install:
     - CMAKE_ARGS=""
     - |
         if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-            sudo add-apt-repository -y ppa:george-edison55/cmake-3.x # want cmake newer than in ubuntu 14.04
             sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test # for gcc-6
             sudo apt-get update
-            sudo apt-get install -y cmake libperl-dev libenchant-dev qtmultimedia5-dev libqt5webkit5-dev libqt5svg5-dev libqt5x11extras5-dev libphonon4qt5-dev libphonon4qt5experimental-dev g++-6
```

Comment:
why?

Reply:
the xenial image has a current cmake image so we no longer need to install it

Reply:
cmake is not installed by default in the image of ubuntu